core/base/inc/Rtypes.h

```diff
@@ -320,27 +322,27 @@ public: \
    virtual_keyword void Streamer(TBuffer &R__b) overrd { ::ROOT::Internal::DefaultStreamer(R__b, name::Class(), this); }
 
 #define ClassDef(name,id) \
-   _ClassDefOutline_(name,id,virtual,)   \
+   _ClassDefOutline_(name,id,virtual,) \
    static int DeclFileLine() { return __LINE__; }
```

Comment:
```suggestion
 #define ClassDef(name,id)                             \
    _ClassDefOutline_(name,id,virtual,)             \
   static int DeclFileLine() { return __LINE__; }
```
is likely more consistent with the intent/rest-of-code (and in my opinion somewhat easier to read).

Reply:
@pcanal I rebase code and adjust `Rtypes.h` format